content/en/docs/instrumentation/python/exporters.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to
```

Comment:
```suggestion
In order to visualize and analyze your telemetry you will need to
```

Reply:
Good catch, this was initially because I was going to include the prometheus exporter, but I decided not to keep it and missed this bit.